cli/coredump/coredump.go

```diff
@@ -0,0 +1,130 @@
+package coredump
+
+import (
+	"bytes"
+	"context"
+	"embed"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+
+	"github.com/codeclysm/extract"
```

Comment:
Why not using std go libraries  
"archive/tar"
"compress/gzip" ?

Reply:
I obviously can, but it will take more code and that lib already implements it. If u think that it will be more right to write these functions in file, i can change it.

Reply:
@LeonidVas your opinion?

Reply:
Ok, wrote function in file